lib/backend/services/auth_cognito_service.dart

```diff
@@ -266,7 +266,7 @@ class CognitoClient extends AuthenticationService {
   @override
   Future<SignOutResult> logOut() async {
     try {
-      await aws.FlutterAwsAmplifyCognito.signOut();
+      await aws.FlutterAwsAmplifyCognito.signOutGlobally();
```

Comment:
Why? This logs out the user from all sessions across devices. I guess we don't need this here.